src/content-helper/dashboard-page/components/posts-table/component.tsx

```
@@ -49,8 +53,9 @@ const PostInfo = ( { post }: { post: HydratedPost } ): React.JSX.Element => {
 			<div className="post-details">
 				<div className="post-title">
 					{ post.title.rendered !== ''
-						? post.title.rendered
-						: __( '(no title)', 'wp-parsely' ) }
+						? <div dangerouslySetInnerHTML={ { __html: post.title.rendered } } />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)